test/stream/api/sink/statistics-total-001-ok/001-statistics-total/expected-response.json

```diff
@@ -0,0 +1,28 @@
+{
+  "total": {
+    "firstRecordAt": "2000-01-20T00:00:00.000Z",
+    "lastRecordAt": "2000-01-21T00:00:00.000Z",
+    "recordsCount": 1,
+    "uncompressedSize": 1
+  },
+  "levels": {
+    "local": {
+      "firstRecordAt": "2000-01-20T00:00:00.000Z",
+      "lastRecordAt": "2000-01-21T00:00:00.000Z",
+      "recordsCount": 1,
+      "uncompressedSize": 1
+    },
+    "staging": {
+      "firstRecordAt": "0001-01-01T00:00:00.000Z",
+      "lastRecordAt": "0001-01-01T00:00:00.000Z",
```

Comment:
`null` will be better for UI team, but we can fix it in another PR (`IsZero()`).